node_modules/oae-core/tenantlandingpage/js/tenantlandingpage.js

```diff
@@ -0,0 +1,134 @@
+            // full control of the URL hash and adding one into the URL ourself would interfere with that
+            // @see https://github.com/oaeproject/3akai-ux/issues/3872
+            query = query.replace(/#/g, '');
+            window.location = '/search/' + oae.api.util.security().encodeForURL(query);
+            return false;
+        });
+    };
+
+    /**
+     * Set up the videos in the configured page structure. A video will initially show a placeholder
+     * image, which will be replaced by the actual video when the play button is clicked
+     */
+    var setUpVideo = function() {
+        // Ensure that the play icon is correctly positioned at all times
+        repositionPlayVideo();
+        $(window).on('resize', repositionPlayVideo);
+
+        // Replace the placeholder image with the actual video when the play button is clicked
+        $('.tenantlandingpage-video-launch').on('click', function() {
+            var $videoContainer = $(this).parent();
+            var url = $(this).attr('data-url');
+            // Detect YouTube videos and automatically play them when the play button is clicked
+            if (url.indexOf('youtube') !== -1) {
+                var youtubeId = $.url(url).param('v');
+                url = '//www.youtube.com/embed/' + youtubeId + '?rel=0&autoplay=1&showinfo=0&modestbranding=1';
+            // Other videos are embedded as is
+            } else {
+                url = url.replace(/http[s]?:/, '');
+            }
+
+            // Render the video
+            oae.api.util.template().render($('#tenantlandingpage-video-template'), {'videoURL': url}, $videoContainer);
+        });
+    };
+
+    /**
+     * Position the video play button to ensure that it's vertically centered within the video placeholder.
+     * Because the different blocks on the same row are always equally high, there is unforunately no cross-browser
+     * CSS-only solution for this
+     */
+    var repositionPlayVideo = function() {
+        // Reposition the play button for all videos
+        var $videos = $('.tenantlandingpage-block-video');
+        if ($videos.length) {
+            var visible = false;
+            $videos.each(function(videoIndex, video) {
+                var $video = $(video);
+                var height = $video.closest('.tenantlandingpage-row').height();
+                if (height) {
+                    // Make the play button as high as its container, so it centers vertically
+                    $video.css('height', height  + 'px');
+                    // The play button is hidden initially to avoid the button from jumping
+                    // when it is repositioned
+                    $video.show();
+                    visible = true;
+                }
+            });
+            // If none of the videos are currently visible, the page is still in the process
+            // of loading and heights cannot be calculated. Try again later
+            if (!visible) {
```

Comment:
I *think* this might no longer be needed as the container page should now always be visible when this video loads